submitted_models/coro_karen_sensor_config_1/launch/spawner.rb

```diff
@@ -0,0 +1,109 @@
+    <world>#{_worldName}</world>
+    <is_performer>true</is_performer>
+    <sdf version='1.6'>
+    <include>
+      <name>#{_name}</name>
+      <uri>#{_modelURI}</uri>
+      <!-- Ackermann Steering -->
+      <plugin filename="libackermannlocal.so"
+              name="ignition::gazebo::systems::AckermannSteering">
+        <left_joint>front_left_wheel_joint</left_joint>
+        <left_joint>rear_left_wheel_joint</left_joint>
+        <right_joint>front_right_wheel_joint</right_joint>
+        <right_joint>rear_right_wheel_joint</right_joint>
+        <left_steering_joint>front_left_wheel_steering_joint</left_steering_joint>
+        <right_steering_joint>front_right_wheel_steering_joint</right_steering_joint>
+        <wheel_separation>0.65</wheel_separation>
+        <kingpin_width>0.44</kingpin_width>
+        <steering_limit>0.187</steering_limit>
+        <wheel_base>0.93</wheel_base>
+        <wheel_radius>0.175</wheel_radius>
+        <topic>/model/#{_name}/cmd_vel_relay</topic>
+        <min_velocity>-1.0</min_velocity>
+        <max_velocity>1.0</max_velocity>
+        <min_acceleration>-3</min_acceleration>
+        <max_acceleration>3</max_acceleration>
+      </plugin>
+      <!-- Publish robot state information -->
+      <plugin filename="libignition-gazebo-pose-publisher-system.so"
+        name="ignition::gazebo::systems::PosePublisher">
+        <publish_link_pose>true</publish_link_pose>
+        <publish_sensor_pose>true</publish_sensor_pose>
+        <publish_collision_pose>false</publish_collision_pose>
+        <publish_visual_pose>false</publish_visual_pose>
+        <publish_nested_model_pose>#{$enableGroundTruth}</publish_nested_model_pose>
+        <use_pose_vector_msg>true</use_pose_vector_msg>
+        <static_publisher>true</static_publisher>
+        <static_update_frequency>1</static_update_frequency>
+      </plugin>
+      <!-- Battery plugin -->
+      <plugin filename="libignition-gazebo-linearbatteryplugin-system.so"
+        name="ignition::gazebo::systems::LinearBatteryPlugin">
+        <battery_name>linear_battery</battery_name>
+        <voltage>12.694</voltage>
+        <open_circuit_voltage_constant_coef>12.694</open_circuit_voltage_constant_coef>
+        <open_circuit_voltage_linear_coef>-3.1424</open_circuit_voltage_linear_coef>
+        <initial_charge>78.4</initial_charge>
+        <capacity>78.4</capacity>
+        <resistance>0.061523</resistance>
+        <smooth_current_tau>1.9499</smooth_current_tau>
+        <power_load>9.9</power_load>
+        <start_on_motion>true</start_on_motion>
+      </plugin>
+      <!-- Gas Sensor plugin -->"
+      <plugin filename="libGasEmitterDetectorPlugin.so"
+        name="subt::GasDetector">
+        <topic>/model/#{_name}/gas_detected</topic>
+        <update_rate>10</update_rate>
+        <type>gas</type>
+      </plugin>
+      <!-- Wheel slip -->
+      <plugin filename="libignition-gazebo-wheel-slip-system.so"
```

Comment:
@azeey , can you check these wheel slip parameters and trying driving the robot?

Reply:
The turning radius and velocity limits are consistent with the specification.

I don't think WheelSlip (the plugin and the `<slip>` values in `model.sdf`) is needed for Ackermann steering vehicles. I have tested it without WheelSlip and the vehicle behaved virtually the same. I would actually recommend removing WheelSlip because it has disadvantages when driving laterally on steep slopes.

Reply:
I tried driving it around in simple_cave_02 with the WheelSlip plug-in removed, but it slipped sideways even on flat areas.  I didn't get as far as slopes.  Is that expected?

Reply:
No, I wouldn't expect that. Did you also remove the `<slip1>` and `<slip2>` elements from `model.sdf`?

Reply:
Thank you, that was the issue.  I had not removed those.  It seems to behave fine with the slip1/2 and WheelSlip plugin removed.

Reply:
Removed in e014c78